tests/cmake/run_test.sh

```diff
@@ -15,11 +15,17 @@ OUTPUT="$4"
 
 $BINARY $PRM >${OUTPUT} 2>&1
 ret=$?
+
 if [[ ( "$EXPECT_FAIL" == "0" && "$ret" == "0" ) 
 	    ||
       ( "$EXPECT_FAIL" == "1" && "$ret" != "0" ) ]]
 then
+  # success: ASPECT succeeded (or failed if we expect it to fail)
   exit 0
 fi
 
+# ASPECT failed (or succeeded even though it should not have), so report
+# failure. Before we do so, dump the screen-output to the screen (into
+# stderr):
+cat ${OUTPUT} 2>&1
```

Comment:
Isn't this exactly wrong, putting things from stderr to stdout? The redirection as shown makes no difference because `cat` outputs to stdout.